# RobotState.msg
string robot_name
string current_task
bool is_interacting
